test/unit-test/mqtt_agent_utest.c

```diff
@@ -230,6 +239,49 @@ static void setupAgentContext( MQTTAgentContext_t * pAgentContext )
     pAgentContext->mqttContext.nextPacketId = 1U;
 }
 
+static MQTTStatus_t MQTT_Init_stub( MQTTContext_t * pContext,
+                                    const TransportInterface_t * pTransport,
+                                    MQTTGetCurrentTimeFunc_t getTimeFunc,
+                                    MQTTEventCallback_t userCallback,
+                                    const MQTTFixedBuffer_t * pNetworkBuffer )
+{
+    pContext->connectStatus = MQTTNotConnected;
+    pContext->transportInterface = *pTransport;
+    pContext->getTime = getTimeFunc;
+    pContext->appCallback = userCallback;
+    pContext->networkBuffer = *pNetworkBuffer;
+    pContext->nextPacketId = 1;
+    return MQTTSuccess;
+}
+static void setupAgentContext_mqttEventCallback( MQTTAgentContext_t * pAgentContext )
```

Comment:
Uncrustify for spacing
```suggestion

static void setupAgentContext_mqttEventCallback( MQTTAgentContext_t * pAgentContext )
```